apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/MetadataFixtureBase.cs

```diff
@@ -94,6 +92,9 @@ protected MetadataFixtureBase()
             emulatorProcess.Start();
             emulatorProcess.BeginOutputReadLine();
             emulatorProcess.BeginErrorReadLine();
+
+            // Wait 1 second so the server can start before any tests are run.
+            Thread.Sleep(1000);
```

Comment:
Is there a less flaky way of doing this, e.g. until we've seen a particular piece of expected output? This is better than nothing, but it would be nice to be even more reliable.